proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+
+#### Updating Proposal Statuses
+
+If the author of a particular proposal wants to update the status of their proposal, they should either comment on the issue and tag an HF Technical Track member with a note regarding what status they would like to update the proposal to. It is good to be redundant and raise the question in our other fora as well: Slack and Discourse. However, Github should be the first choice. 
+
+### Formal presentation 
+
+After a proposal is merged, the author will present KPI's and deliverables that define the proposal to the HF Technical track. During the next available Technical Track meeting after acceptance, the author or designated lead (as defined by the proposal) should give a short 5-10min presentation consisting of the following: 
+
+1. **High level summary:** the author will present the high-level abstract for what the proposal intends to accomplish. 
+2. **Timeline:** In what timeline, and should outline possible hurdles and risks that affect the timeline. 
+3. **Resources:** What resources are required in order to take the project from start to completion.
+4.  **Contact:** Who is going to update the HF Technical Track members, and on what schedule.
+
+Once the formal presentation is given, the HF Technical Track members will track the progress over time at our ongoing meetings. This also gives the track members time to introduce themselves and and we can all get to know each other. Currently, HF Technical Track meetings are held weekly on Tuesdays at 11:30am-12:30pm UTC-5 (EST). 
+
+## Structure of the process
+
+The HIP process involves the following parties:
+
+1. The HIP Authors
+2. Relevant stakeholders affected by the change
+3. The HF Technical Track members
+
+### The HIP Authors
+
+Authors are responsible for building consensus within the community and documenting dissenting opinions before the HIP is officially discussed by the HIP HF Technical Track . Their goal is to convince the HF Technical Track that their proposal is useful and addresses pertinent problems in the Haskell ecosystem as well as interactions with already existing features. Authors can change over the life-cycle of the HIP.
+
+### The HF Technical Track
+
+The HF Technical Track is an experienced group of people with knowledge of the Haskell ecosystem, responsible for the strategic technical direction on behalf of the Haskell Foundation. Members are tasked with
+
+  - communicating with the community
+  - weighing in pros and cons of every proposal
+  - accepting, postponing or rejecting the proposal.
+
+HF Technical Track members should be either individuals responsible for a specific part of the Haskell ecosystem, or contributors and committers to parts of it. The members are selected by the HF CTO based on their expertise and implication in the community.
```

Comment:
Did you mean "implication"? Perhaps "reputation" would be better?

I may have complained about this previously, but I'm now OK with you, @emilypi, unilaterally selecting the HFTT. We serve at your pleasure. But I do think there should be a process by which a community member can apply to join. Having open applications works against a situation where you choose only people close to you -- not because of bias, but because those are the people you're aware of.

Is this list of people up-to-date? Some names here are not regular attendees of our meetings and may be surprised to be listed. Best to double-check with everyone. (I explicitly agree to serve.)

Reply:
> Did you mean "implication"? Perhaps "reputation" would be better?

In a sense, implication is both the correct word and too strong a word here. We don't want the HFTT to be composed exclusively of Haskell titans, which sets the bar extremely high and gives the wrong impression. I'd be perfectly fine with having enthusiastically dedicated students, as well as representatives from industry, in addition to those big Haskell names. But implication sounds a bit... political. Here, we'd want to say something broader, like taking a sample of the community.